s64da_benchmark_toolkit/prepare.py

```diff
@@ -176,18 +176,32 @@ def _load_pre_schema(self, conn):
                 conn.cursor.execute(pre_schema_file.read())
 
     def _load_license(self, conn):
-        license_path = '/s64da.license'
         try:
-            conn.cursor.execute(f'select swarm64da.load_license(\'{license_path}\')')
-            print(f'Loading S64 DA License {license_path}')
+            conn.cursor.execute(f'select swarm64da.show_license()')
+        except errors.InvalidSchemaName:
+            print(f'Selected schema is not for S64 DA. Skip checking license.')
```

Comment:
That message is misleading, there is no schema selection.

Reply:
InvalidSchemaName is the error type (according to experiments), when the psql function is not known. This is the case when the extension is not loaded, which in turn is only the case when the toolkit is run on a native schema.
Would the error message "S64 DA extension not loaded" be clearer ?

Reply:
Maybe make it even more vague. Like "S64 DA seems not being loaded." or so, just trying to think about situations where the extension is there but the schema is not.